source/Ship.cpp

```diff
@@ -3462,9 +3462,10 @@ bool Ship::CanBeFlagship() const
 
 
 
+// Account for inertia reduction
 double Ship::Mass() const
 {
-	return carriedMass + cargo.Used() + attributes.Mass();
+	return carriedMass + cargo.Used() + attributes.Mass() / (1 + attributes.Get("inertia reduction"));
```

Comment:
Can you assure that `Ship::Mass()` is only ever used to check the *inertial* mass and might not be used, for example, to check *thermal* mass (before adding `"heat capacity"`).

Also, the way you've done this division, the attribute will only reduce the inertial mass of the ship itself and not include any cargo or carried ships. Maybe it shouldn't affect those, but you should at least make that clear in a comment.

Reply:
I think it'd be much better if you added an `InertialMass()` method that returns this modified value and is explicitly only used in movement calculations, instead of modifying the `Mass()` method which may be fine right now, but may not remain fine forever.